README.md

```diff
@@ -1,5 +1,131 @@
-# Minishift Fedora VM
+<a name="creating-a-minishift-fedora-iso"></a>
+# Minishift Fedora ISO
 
 This repository contains all the instructions and code to build a Live ISO based on Fedora
 which can be used by [minishift](https://github.com/minishift/minishift) as an alternative to
-the boot2docker and CentOS ISO.
+the boot2docker ISO.
```

Comment:
Isn't this should be CentIOS ISO ? This line is telling me that we have b2d ISO still as default.

Reply:
I read the same. We soon do not have b2d anymore...